cli/wallet/nep11.go

```diff
@@ -254,6 +255,7 @@ func signAndSendNEP11Transfer(ctx *cli.Context, c *client.Client, acc *wallet.Ac
 	if err != nil {
 		return cli.NewExitError(err, 1)
 	}
+	tx.NetworkFee += int64(sysgas)
```

Comment:
But why network fee? We should add it to the system fee.

Reply:
Yep, fixed.